pkg/test/istioio/script.go

```diff
@@ -458,7 +456,7 @@ type snippetInfo struct {
 	outputSnippet  bool
 	commandLines   []string
 	expectedOutput []string
-	verifyOutput   verifier
+	verifyOutput   []verifier
```

Comment:
It works with my test (I'll take a look if this breaks other tests)
The use-case here is super common: We have a snippet output with multiple lines and each of them has some randomness in it (pods + UUID name), the obvious approach for a verifier would be a regular expression for each line.
The way verifier is currently implemented (before that PR) forces a test developer to create a very large regex to match multiple lines of an input.
With the proposed/new approach, a test developer to  could provide multiple regexes that are each applied to entire input. Any regex match failing will fail the test.

Reply:
Sure, my only question is: how to cleanly provide multiple inputs to multiple verifiers (within the snippet syntax). 

Would it make sense to combine regexes with the token-based verifier? Where instead of just providing wildcards (`?`), you can specify regexes for fields?

Reply:
I think I see what you're meaning:
```
                        info.expectedOutput = append(info.expectedOutput, line)
```
Makes the expectedOutput (what we call "inputs" in these comments), to be put in a slice that I then assign on a one-entry-of-slice-per-one-verifier basis.

This means my PR inelegantly forces (well, it expects, it doesn't force anything, which is a bug ;-) ) verifier to be followed by a single line value for that verifier.

What I like with this per-line approach is that it would work with an unsorted output, but I think I really got your point, and it is right, let me re-work that as a token regex matcher. Definitely more elegant.

Reply:
Yeah exactly. I wouldn't create another verifier, I'd just re-work the existing token verifier to accept regexes. There are only a couple existing tests ... if they had to be tweaked, it would probably be fine.

Reply:
FYI, I created this issue to track cleanup of the token matching: 

https://github.com/istio/istio/issues/18297